### ISSUE TITLE
New Icon

### ISSUE THREAD

**Issue by Vittelius** [patrickjane/ut-plants#1]
The current icon doesn't really follow the Ubuntu Touch Design Guidelines.
I have created a version that does and humbly submit it hereby:

![Plants_icon](https://user-images.githubusercontent.com/3718818/174137720-c08bee17-2212-4ca8-beb7-4e4956352a6f.svg)

Also here is a version, that doesn't strictly follow the guides because the fold isn't straight:
![Plants_icon2](https://user-images.githubusercontent.com/3718818/174137878-aba9f76b-a932-4b16-ac49-e2471df48bba.svg)

**Comment by patrickjane** [comment]
Hi, thanks for your feedback. So basically I shouldn't omit the fold effect? Maybe I am reading it wrong, but app icon design page does not require to have it, right?

Maybe like this?

![logo2](https://user-images.githubusercontent.com/1404715/174277029-54498c40-1a99-4674-9e7b-7781824a6700.png)

**Comment by Vittelius** [comment]
You are right, the fold is not required. But I still feel your icon is slightly off. I think it's the gradient. I think the colors you've picked are too far apart. (That's why so many icons have that fold: If you follow the guidelines the different colors are barely perceptible. My design has a gradient, but you wouldn't know it unless you gave it a second look)
But then again the guidelines are only that: guidelines. I doubt many people will even notice, there are certainly worse offenders on the store and nobody's complaining.
Your icon just felt (like I wrote) slightly off. 99% there and I couldn't put my finger on what the missing 1% was. So I tried my hand at it. And at that point I might as well throw my result out there (Not saying my design is perfect, just my best effort).

**Comment by ghost** [comment]
Hi, I am quilty weavy. Patrickjane, to be honest, the icon that vittelius has submitted  looks better. Why don't you try replacing the current icon with their icon and make the iconic phase of your app better ? It will increase the app's productivity. Besides, your current plant icon looks similar to the svg icon in this link :- https://www.onlinewebfonts.com/icon/40025 . Also, the icon they have posted is a svg icon, which is much better than a png icon

**Comment by patrickjane** [comment]
> You are right, the fold is not required. But I still feel your icon is slightly off. I think it's the gradient. I think the colors you've picked are too far apart. (That's why so many icons have that fold: If you follow the guidelines the different colors are barely perceptible. My design has a gradient, but you wouldn't know it unless you gave it a second look) But then again the guidelines are only that: guidelines. I doubt many people will even notice, there are certainly worse offenders on the store and nobody's complaining. Your icon just felt (like I wrote) slightly off. 99% there and I couldn't put my finger on what the missing 1% was. So I tried my hand at it. And at that point I might as well throw my result out there (Not saying my design is perfect, just my best effort).

I was just curious about the guidelines, since nothing I can read there explicitly states what *should* be done, only what *some apps do*. That's why I asked what you think is missing.
I think the part about the gradient might be there to tell people not to do red to blue gradients, after all I think "the change is not too dramatic" is a pretty subjective view. 

But actually I like your gradient better than mine. I also noticed I failed to reuse the colors I am using in the app, which would be a reason alone to adjust the app icon.
However what I dislike about your version is both the leaf as well as the leaf coloring - with the latter being also non guidelines compliant btw ;) So I'd rather not like to use it.

However, seeing that you both seem to be pretty unhappy with my app icon, I don't want to get into the way of changing it. Maybe we can do some kind of compromise? Use your background and my leaf? Or maybe one of those:

https://www.onlinewebfonts.com/icon/40103

https://www.onlinewebfonts.com/icon/40095

https://www.onlinewebfonts.com/icon/40073

My current app icon follows what I did with my other apps: https://open-store.io/?sort=relevance&search=author%3Apatrickjane So maybe there should be some adjustments to those icons too.

**Comment by ghost** [comment]
I have found a better plant icon in this link :- https://www.onlinewebfonts.com/icon/412871 . Why dont you use this? Vitellius's folded background also looks good so use that background with the icon I provided you in the above link. Also, please convert your icon into a svg icon

**Comment by patrickjane** [comment]
This one does not match my taste :/ I don't like it.

**Comment by ghost** [comment]
Then how about this one :- https://www.onlinewebfonts.com/icon/535133

**Comment by patrickjane** [comment]
So you *really* don't like any of the pictures I suggested? The one you posted is pretty close to https://www.onlinewebfonts.com/icon/40095

**Comment by ghost** [comment]
Ok, then go ahead. But the link I posted looks smooth and elegant.
